modules/continuous-delivery/pages/configuring-the-platform.adoc

```diff
@@ -68,4 +68,18 @@ NOTE: the Bonita version is displayed next to the runtime name.
 . Make sure the job was successful:
 image:def-deploy-job-result.png[JobResult]
 
+== Default Deploy configuration for UIB applications
 
+This job allows you to define the default build parameters i.e. the repository URL, the branch, and environment to build to avoid having to re-enter the same parameters in the deploy-uib job.
```

Comment:
We should mention that the job related to UIB is only useful for version of Bonita >= 2024.3/10.2